tests/wrap_malloc.cpp

```diff
@@ -20,15 +21,15 @@ void * __attribute__((noinline)) malloc(size_t size)
 {
 	++totalAllocated;
 	void *ptr = emscripten_builtin_malloc(size);
-	printf("Allocated %zu bytes, got %p. %d pointers allocated total.\n", size, ptr, totalAllocated);
+	emscripten_log(EM_LOG_CONSOLE, "Allocated %zu bytes, got %d. %d pointers allocated total.\n", size, ptr, totalAllocated);
```

Comment:
Prefer `emscripten_console_logf` I think...  IIUC `emscripten_log` does more like a swis army knife where as `emscripten_console_xxx` just write to the console..

Reply:
Done. Also in the existing test `test_dlmalloc` that was similarly modified recently.